pex/venv/virtualenv.py

```diff
@@ -224,22 +222,18 @@ def create_atomic(
     def __init__(
         self,
         venv_dir,  # type: str
-        python_exe_name="python",  # type: str
```

Comment:
Again not relevant to this fix, but noted as a point of potential non-robustness for exaotic scenarios (pypy) and so simplified.

Reply:
Ok, this "fix" triggered infinite recursion in the magic `__pex__` import hook package under Python 2.7 only. I'll revert.